infra/gcb/build_and_run_coverage.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/python2
+
+"""Starts and runs coverage build on Google Cloud Builder.
+
+Usage: build_and_run_coverage.py <project_dir>
+"""
+
+import datetime
+import os
+import sys
+
+import build_projects
+
+BUILD_TIMEOUT = 10 * 60 * 60
+
+CONFIGURATION = ['FUZZING_ENGINE=libfuzzer', 'SANITIZER=profile']
+
+SANITIZER = 'coverage'
+
+def usage():
+  sys.stderr.write(
+    "Usage: " + sys.argv[0] + " <project_dir>\n")
+  exit(1)
+
+def get_build_steps(project_dir):
+  project_name = os.path.basename(project_dir)
+  project_yaml = build_projects.load_project_yaml(project_dir)
+  dockerfile_path = os.path.join(project_dir, 'Dockerfile')
+  name = project_yaml['name']
+  image = project_yaml['image']
+
+  ts = datetime.datetime.now().strftime('%Y%m%d%H%M')
+
+  build_steps = [
+      {
+          'args': [
+              'clone', 'https://github.com/google/oss-fuzz.git',
+          ],
+          'name': 'gcr.io/cloud-builders/git',
+      },
+      {
+          'name': 'gcr.io/cloud-builders/docker',
+          'args': [
+              'build',
+              '-t',
+              image,
+              '.',
+          ],
+          'dir': 'oss-fuzz/projects/' + name,
+      },
+  ]
+
+  env = CONFIGURATION[:]
+  out = '/workspace/out/' + SANITIZER
+  stamped_name = name + '-' + SANITIZER + '-' + ts
+  zip_file = stamped_name + '.zip'
```

Comment:
Actually, I should upload report as is, without zipping.

Reply:
+1